### DISCUSSION TITLE
What is "accented" state?

### DISCUSSION THREAD

**Discussion by gmaclellan-ecisolutions** [jseidl/magic-areas#357] [General]
One of the [area states](https://github.com/jseidl/hass-magic_areas/wiki/Area-State) is "accented": what is the intended purpose of this, and what are people doing with it? What kind of sensors are used to indicate it?

**Comment by jseidl** [comment]
Hi! This is for when you want to get the area in a "mood" state. I personally use for when my TV is on, to turn on my accent lights on the sides of the tv and turn off all the other lights. I use a smart outlet's power consumptions sensor with a Threshold sensor to indicate whether the TV is active and use that Threshold sensor as my accented state entity.
